variables.tf

```
@@ -8,21 +8,27 @@ variable "resource_prefix" {
   }
 }
 
-variable "deployment_name" {
-  description = <<EOF
-(optional) Databricks deployment name for the workspace, if any, as explained in
-https://registry.terraform.io/providers/databricks/databricks/latest/docs/resources/mws_workspaces#deployment_name
-  EOF
-  type        = string
-  nullable    = true
-  default     = null
+variable "workspaces" {
+  description = "Databricks workspaces name(s), to use with the deployment (optional) and workspace names. Will deploy as many workspaces as given names"  
```

The last part of the first sentence sounds a bit strange to me, some rephrase that came to mind including some extra info:
```suggestion
  description = "Databricks workspace name(s). Will deploy one workspace per key, and will set each name as workspace name - and optionally as deployment name, if add_deployment_name is true. NOTE: all workspaces will share the same Databricks configurations for credentials, networks and storage, i.e. they will share crossacount role, subnets, S3 buckets, etc. If this is not the intended result, call the module several times instead"
```